browser/ui/color/sources.gni

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_browser_ui_colors_sources = [
```

Comment:
nit: I think it should be `brave_browser_ui_color_mixers_sources` (and var below fixed accordingly).

Reply:
Renamed.